x-pack/plugins/cases/README.md

```diff
@@ -140,6 +171,57 @@ Arguments:
 UI component:
 ![Recent Cases Component][recent-cases-img]
 
+### hooks
+
+
+#### getUseCasesAddToNewCaseFlyout
+
+Returns an object containing two methods: `open` and `close` to either open or close the add to new case flyout.
+You can use this hook to prompt the use to create a new case with some attachments directly attached to it. e.g.: alerts or text comments.
```

Comment:
Looks like you still have `the use` in this sentence.

Reply:
Thanks. Fixed.